src/main/drivers/lights_hal.c

```diff
@@ -0,0 +1,30 @@
+#include "drivers/lights_hal.h"
```

Comment:
I don't like the name `lights_hal`. We use `_hal` suffix in drivers that depend on ST's HAL library. This is clearly not the case. Maybe use `lights_io` name here?

Reply:
As you like. I thought it would be appropriate as it is already used for WS2811 lights: `light_ws2811strip_hal.c`.

Reply:
Exactly my case - `light_ws2811strip_hal.c` is driver designed to work with ST's HAL library. That's why we have `light_ws2811strip_stdperiph.c` next to is (using ST's StdPeriph library). We don't depend on ST's CPU-support libraries, so let's rename.